keps/sig-node/3695-pod-resources-for-dra/kep.yaml

```diff
@@ -18,12 +18,12 @@ see-also:
 replaces: []
 
```

Comment:
Does line 7 need to be updated?
`status: provisional` => `status: implemented`

Reply:
It needs to be "implementable". I believe "implemented" is set once it's done, i.e. stable.